pkg/capabilities/consensus/ocr3/ocr3_common-schema.json

```diff
@@ -27,6 +27,14 @@
       },
       "additionalProperties": false,
       "required": ["Report", "Context", "Signature", "ID"]
+    },
+    "encoder": {
+      "type": "string",
+      "enum": ["EVM"]
+    },
+    "encoder_config": {
+      "type": "object",
+      "additionalProperties": true
```

Comment:
Is this meant to be `any`? Why switch from https://github.com/smartcontractkit/chainlink-common/pull/743/files#diff-7b48ddffa1322b95afbcda50d5bd195e0d59e5069ed0a1ef651e3c31dd122b81L71-L84 ?

Reply:
@cedric-cordenier pointed out that's only for EVM.  It should work for all encoders.